provisioning/local/update-env.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+ENV_FILE=".env"
+INSERT_MODE=prepend
+VERBOSE=false
+
+help () {
+  echo "Syntax: update-env.sh [-v] [-a] [-e ${ENV_FILE}]"
```

Comment:
Koukam ze si tu uplne vyhodil tu volbu cloudu (aws vs azure). Uz jsem taky nekde narazil na projekt, kde to nebylo potreba rozlisovat (protoze se oboje provisionovalo najednou, nebylo potreba prepinat), ale rozhodl jsme se nakonec tam ten argument nechat proste z duvodu, ze se to na vsehc projektech pak pouziva stejne. I kdyz v tu chvili ten argument fakticky nic nemeni, nemusis premyslet, v cem se tenhle projekt lisi, ze se to pousti jinak. Dovedl bych si dokonce predstavit, ze nekdy dal v budoucnu by tenhle skript nemusel byt u kazdyho projektu, ale separatni, 1x lokalne nainstalovany.

Reply:
Jo s tím souhlasím, že by ten skript měl být v nějaké libce.
Stejně tak modul pro založení bucketu/abs containeru pro logy. Co jsem teď koukal různě napříč repos, tak je to každý pes jiná ves.